packages/react-native-bpk-component-calendar/stories.js

```diff
@@ -424,7 +424,11 @@ storiesOf('react-native-bpk-component-calendar', module)
             minDate={minDate}
             maxDate={maxDate}
             colorBuckets={[
-              colorBucket(colorSagano, dateMatchers[selectedValue].descriptor),
+              colorBucket(
+                colorSagano,
+                dateMatchers[selectedValue].descriptor,
+                'light',
```

Comment:
Why adding the default value?

Reply:
The default value is undefined, by the style should be light on `colorSagano`

Reply:
There is a bit of mismatch in how Android and iOS deal with this then, Android will use the best value based on the bucket colour when it's undefined.

Reply:
Oh right ok - we should do the same for each platform I guess

Reply:
What algorithm do you use to determine luminance of the background color?